estuary/values.yaml

```diff
@@ -0,0 +1,17 @@
+# make a dependency when kdp is published
+presto:
+  url: "http://kdp-kubernetes-data-platform-presto.kdp:8080"
+
+image:
+  repository: smartcitiesdata/apps/estuary
```

Comment:
I don't think this is the name of the dockerhub repository.  See what other apps do here.

Reply:
I was not sure about that because all the apps are using smartcitiesdata/{app-name} even though they are inside umbrella, my guess is it should be smartcitiesdata/smartcitiesdata pointing to the top. Since I was not sure so I've put that.